gobreaker_test.go

```diff
@@ -211,6 +239,61 @@ func TestCustomCircuitBreaker(t *testing.T) {
 	assert.Equal(t, StateChange{"cb", StateHalfOpen, StateClosed}, stateChange)
 }
 
+func TestMultiStepBreaker(t *testing.T) {
```

Comment:
I think it's better to rename `TestMultiStepBreaker` to `TestTwoStepCircuitBreaker`.